_includes/social-links.html

```diff
@@ -1,35 +1,7 @@
-<p class="control social-link-control">
-  <a href="https://twitter.com/{{ site.twitter_username }}" target="_blank" title='Twitter' class="is-size-5 rnw-social-link">
-    <i class="fab fa-twitter"></i>
-  </a>
-</p>
-<p class="control social-link-control">
-  <a href="https://github.com/{{ site.github_username }}" target="_blank" title='GitHub' class="is-size-5 rnw-social-link">
-    <i class="fab fa-github"></i>
-  </a>
-</p>
-<p class="control social-link-control">
-  <a href="https://www.youtube.com/channel/{{site.youtube_channelname}}" target="_blank" title='YouTube' class="is-size-6 rnw-social-link">
-    <i class="fab fa-youtube"></i>
-  </a>
-</p>
-<p class="control social-link-control">
-  <a href="https://www.facebook.com/{{ site.facebook_username }}" target="_blank" title='Facebook' class="is-size-5 rnw-social-link">
-    <i class="fab fa-facebook"></i>
-  </a>
-</p>
-<!-- <p class="control social-link-control">
-  <a href="https://{{ site.slack_username }}.slack.com/" target="_blank" title='Slack' class="is-size-5 rnw-social-link">
-    <i class="fab fa-slack"></i>
-  </a>
-</p>
-<p class="control social-link-control">
-  <a href="https://www.meetup.com/{{ site.meetup_username }}" target="_blank" title='Meetup' class="is-size-5 rnw-social-link">
-    <i class="fab fa-meetup"></i>
-  </a>
-</p> -->
-<p class="control social-link-control">
-  <a href="/feed.xml" target="_blank" title='Subscribe' class="is-size-5 rnw-social-link">
-    <i class="fas fa-rss"></i>
-  </a>
-</p>
+{% for contact in site.data.info.contacts %}
```

Comment:
This does make the code terse but not necessarily readable. 
The power of jekyll is in allowing mixing of html with data contents so that we know what the output looks like beforehand without interpreting the code.

I would be fine with this code if above concerns are addressed.